packages/driver/src/cypress/screenshot.coffee

```diff
@@ -16,6 +16,40 @@ defaults = reset()
 
 validCaptures = ["fullPage", "viewport", "runner"]
 
+normalizePadding = (padding) ->
+  if not padding
+    padding = 0
+
+  if _.isArray(padding)
+    # CSS shorthand
+    # See: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases
+    switch padding.length
+      when 1
+        top = right = bottom = left = padding[0]
+      when 2
+        top = bottom = padding[0]
+        right = left = padding[1]
+      when 3
+        top = padding[0]
+        right = left = padding[1]
+        bottom = padding[2]
+      when 4
+        top = padding[0]
+        right = padding[1]
+        bottom = padding[2]
+        left = padding[3]
+      else
+        top = right = bottom = left = 0
```

Comment:
probably should error here instead of zeroing out to handle the condition where length isnt' 1-4

Reply:
or remove this if this case is impossible to get to